README.md

```diff
@@ -5,201 +5,95 @@
 ![CI](https://github.com/aviatesk/JET.jl/workflows/CI/badge.svg)
 [![codecov](https://codecov.io/gh/aviatesk/JET.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/aviatesk/JET.jl)
 
-JET.jl employs Julia's type inference system to detect potential bugs.
+JET.jl employs Julia's type inference system to detect potential bugs and type instabilities.
 
-!!! note
+:bangbang:
     The latest version of JET requires Julia versions **1.8** and higher;
     JET is tested against [the current stable release](https://julialang.org/downloads/#current_stable_release) as well as [nightly version](https://julialang.org/downloads/nightlies/). \
     Also note that JET deeply relies on the type inference routine implemented in [the Julia compiler](https://github.com/JuliaLang/julia/tree/master/base/compiler),
     and so the analysis result can vary depending on your Julia version.
     In general, the newer your Julia version is, the more accurately and quickly you can expect JET to analyze your code,
     assuming the Julia compiler keeps evolving all the time from now on.
 
-!!! tip
-    Hold on tight and fasten your seatbelt while JET is analyzing your code for the first time.
-    Once the caches get accumulated, subsequent analysis will run at ✈ speed.
+## Quickstart
+See more commands, options and explanations in the documentation.
 
-
-## Demo
-
-Say you have this strange and buggy file and want to know where to fix:
-
-> demo.jl
+### Detect type instability with `@report_opt`
+Type instabilities can be detected in function calls using the `@report_opt` macro, which works similar to the `@code_warntype` macro. Note that, because JET relies on Julia's type inference, it cannot "see" what happens on the other side of a type instability.
```

Comment:
It's not clear (to me) what "the other side of a type instability" means here.

Reply:
Rephrased: "Note that, because JET relies on Julia's type inference, if a chain of inference is broken due to dynamic dispatch, then all downstream function calls will be unknown to the compiler, and so JET cannot analyze them."